ice40/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+set(ICESTORM ${symbiflow-arch-defs_SOURCE_DIR}/third_party/icestorm)
+set(ICEBOX ${ICESTORM}/icebox/)
+set(ICEPROG_TOOL ${ICESTORM}/iceprog/iceprog)
+
+define_arch(
+  ARCH ice40
+  YOSYS_SCRIPT
+    "synth_ice40 -nocarry $<SEMICOLON>  ice40_opt -unlut $<SEMICOLON> abc -lut 4 $<SEMICOLON>  opt_clean"
```

Comment:
I think @JohnDMcMaster showed that `synth_ice40 -vpr` was actually enough here now? Let's investigate after landing....